src/OAuth2.php

```diff
@@ -61,6 +62,15 @@ abstract class OAuth2 extends BaseOAuth
      * @since 2.1
      */
     public $validateAuthState = true;
+    /**
+     * If to enable proof key for code exchange (PKCE) support and add
+     * a `code_challenge` and `code_verifier` to the auth request
+     *
+     * @see https://oauth.net/2/pkce/
+     *
+     * @var bool
```

Comment:
```suggestion
     * @var bool Whether to enable proof key for code exchange (PKCE) support and add
     * a `code_challenge` and `code_verifier` to the auth request.
     * @since 2.2.10
     *
     * @see https://oauth.net/2/pkce/
```